docs/users/step-debugging.md

```diff
@@ -1,4 +1,4 @@
-# Step-debugging with ddev and xdebug
+<h1>Step-debugging with ddev and xdebug</h1>
```

Comment:
Just noticed the change to `<h1>` - Interested in what is different between `<h1>` and `#` - seems lower headings are not having to be updated?